contracts/Parameters/ParameterManager.sol

```diff
@@ -115,4 +116,11 @@ contract ParameterManager is Initializable, ParameterManagerStorageV2 {
         nativeWrappedToken = _nativeWrappedToken;
         emit NativeWrappedTokenUpdated(_nativeWrappedToken);
     }
+
+    /// @dev Setter for the native wrapped ERC20 token (e.g. WMATIC)
+    function setFreeReactionLimit(uint256 _reactionLimit) external onlyAdmin {
```

Comment:
We should add a unit test in `ParameterManager.ts` to verify the getting/setting of this param.

Reply:
done